### PR TITLE
Modify the variance of base distribution in RQSpline

### PR DESCRIPTION
**Pull request by alicjapolanska**
Add argument to scale the covariance matrix of base distribution.

### CODE REVIEW

**Comment by kazewong** [comment]
@alicjapolanska In the newest release and #123, the normalizing flow interface received a major rehaul. It is now possible to pass a custom distribution in, including a Gaussian distribution with a different variance.

As I am updating the documentation for the API, here is an example of how you can change the variance of the base distribution:

Consider this example gotten from test_normalizingFlow.py:
```
    key1, rng, init_rng = jax.random.split(jax.random.PRNGKey(0), 3)
    data = jax.random.normal(key1, (100,2))
    num_epochs = 5
    batch_size = 100
    learning_rate = 0.001
    momentum = 0.9
    model = MaskedCouplingRQSpline(2, 4, [32,32], 4 , rng, data_mean = jnp.mean(data, axis=0), data_cov = jnp.cov(data.T))
    optim = optax.adam(learning_rate, momentum)
    train_flow, train_epoch, train_step = make_training_loop(optim)
    rng, best_model, loss_values = train_flow(
        rng, model, data, num_epochs, batch_size, verbose = True
    )
    rng_key_nf = jax.random.PRNGKey(124098)
    model.sample(rng_key_nf, 10000)
```

we can replace line 
```
model = MaskedCouplingRQSpline(2, 4, [32,32], 4 , rng, data_mean = jnp.mean(data, axis=0), data_cov = jnp.cov(data.T))
```

with

```
base_dist = Gaussian(jnp.zeros(n_features), jnp.eye(n_features)*0.5, learnable=False)
model = MaskedCouplingRQSpline(2, 4, [32,32], 4 , rng, data_mean = jnp.mean(data, axis=0), data_cov = jnp.cov(data.T), base_dist=base_dist)
```

Would you mind trying out the new API and let me know whether this works for your use case?
